marketpulse/main/models.py

```diff
@@ -57,6 +57,9 @@ class Contribution(models.Model):
     comment = models.TextField(blank=True, default='')
     availability = models.BooleanField(default=True)
 
+    class Meta:
+        ordering = ['-created_on']
```

Comment:
If we won't to list the contributions based on the latest activity for each contribution, maybe we should switch this to `updated_on`. Worst case scenario will be the same as `created_on`.